src/components/Chip/Chip.tsx

```diff
@@ -6,7 +6,7 @@ import type { KeyboardEvent, MouseEvent } from "react";
 export type Props = {
   lead?: string;
   onClick?: (
-    event: MouseEvent<HTMLDivElement> | { lead: string; cloud: string }
+    event: MouseEvent<HTMLDivElement> | { lead: string; value: string }
```

Comment:
Can you add a test for the `onClick` handler? I'd have expected a failure after this change.